crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -2632,15 +2710,23 @@ impl AuthorityPerEpochStore {
             self.get_reconfig_state_read_lock_guard()
                 .is_reject_all_certs()
         };
+        if !is_reject_all_certs {
+            // At this point, RejectAllCerts is the only state we can advance from.
+            return Ok((lock, false));
```

Comment:
It seemed clearer to leave it this way since computing `is_reject_all_certs` is complicated enough that it deserved getting a variable name. I don't have a strong objection to changing it but wouldn't that mean I have to repeat the same "return" twice? Or am I missing a cleaner way to do it?

Reply:
ok, you're right, nm